documentation/overview.md

```diff
@@ -1002,20 +1002,29 @@ Please note that the denominator we use is the living wage for a single full-tim
 
 ### Overview
 
-* **Analyst & Programmer:** Tina Chelidze
-* **Year(s):** 2021
-* **Final data name(s):** `digital_access_county_2021.csv` and `digital_access_city_2021.csv'
-* **Data Source(s):** Census Bureau's American Community Survey 
-* **Notes:**
-* **Data Quality Index:** `1` means the observation count informing the estimate is greater than or equal to 30. The quality index is a `2` otherwise.
+* **Analyst & Programmer:** Tina Chelidze and Manuel Alcalá Kovalski
+* **Year(s):** 2016, 2017, 2018, 2019, 2021, 2022
+
+* **Final data name(s):** `digital_access_county_2022.csv`, 
+`digital_access_city_2022.csv`, `digital_access_county_all.csv`, 
+`digital_access_city_all.csv`, `digital_access_county_subgroup_all.csv`,   `digital_access_city_subgroup_all.csv`,
+* **Data Source(s):** Census Bureau's 5-year ACS.
+* **Notes:** The unit of analysis for the overall and income subgroup is 
+households, whereas the unit of analysis for the race/ethnicity subgroup is
+people in households. 
+* **Data Quality Index:** `1` means the differnence between the estimate and the
+lower bound estimate is less than 0.1, `2` means the difference is between 0.1
+and 0.2, and `3` means the difference is greater than 0.2. 
 * **Limitations:**
 * **Missingness:** There are no missing county or city level observations.
```

Comment:
This is no longer true, right? (re: missing observations), so let's remove or update